ci/test/lint-fast.sh

```diff
@@ -30,10 +30,14 @@ try cargo --locked about generate ci/deploy/licenses.hbs > /dev/null
 
 
 if [[ "${BUILDKITE_PULL_REQUEST:-true}" != "false" ]]; then
+  # see ./ci/test/lint-buf/README.md
+
   fetch_pr_target_branch
 
+  ci_collapsed_heading "Generate protobuf config"
+  try bin/pyactivate ./ci/test/lint-buf/generate-buf-config.py
+
   ci_collapsed_heading "Lint protobuf"
-  # exceptions can be defined in src/buf.yaml
   COMMON_ANCESTOR="$(get_common_ancestor_commit_of_pr_and_target)"
   try buf breaking src --against ".git#ref=$COMMON_ANCESTOR,subdir=src"
```

Comment:
Consider `--verbose` here . Currently in the CI no output is printed on success, which makes me nervous.

Reply:
✅